cfme/tests/openstack/cloud/test_volumes.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for Openstack cloud volumes"""
+
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils import testgen
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate([OpenStackProvider], scope='module')
+
+pytestmark = [pytest.mark.usefixtures("setup_provider_modscope")]
+
+
+@pytest.fixture(scope='module')
+def volume_size():
+    return 1
+
+
+@pytest.yield_fixture(scope='function')
+def volume(appliance, provider, volume_size):
+    collection = appliance.collections.volumes
+    volume = collection.create(name=fauxfactory.gen_alpha(),
+                               storage_manager='{} Cinder Manager'.format(provider.name),
+                               tenant=provider.get_yaml_data()['tenant'],
```

Comment:
**Optional**

Consider using `data` property here instead of calling get_yaml_data() directly.

```
provider.data.get('tenant')